source/event_loop.c

```diff
@@ -69,6 +67,7 @@ static void s_aws_event_loop_group_shutdown_async(struct aws_event_loop_group *e
 
     struct aws_thread_options thread_options;
     AWS_ZERO_STRUCT(thread_options);
+    thread_options.join_strategy = AWS_TJS_MANAGED;
```

Comment:
This says the threads are managed, but then we are manually twiddling the thread counts, which indicates unmanaged threads. We should pick one, this reads like a bug. Alternatively, we'll need to document it thoroughly, because this is not how we would expect threads to be managed.

Reply:
It's the clean up thread which is managed.  Event loop threads are unmanaged but counted.

Reply:
Oh my bad, thanks for clearing that up!